feat(github-actions-checks): init actions (#4)
sendsay-ru/github-action-checks: README.md

npx github-action-checks
```

> The file `.github.checks.json` will appear in your repository. You can make the necessary changes in it and re-run the utility to make changes in workflow according to the config.<br>So, just edit this file and run again:

Reviewer: ```suggestion
The file `.github.checks.json` will appear in your repository. You can edit it as you like and re-run the utility to update workflow according to the config.<br>So, just edit this file and run again:
```

На твое усмотрение

      ]
    }
  ],
  "showComments": false,
  "node": 16,
  "installCommand": "npm install",
  "buildCommand": ""
}
```

`showComments` - show [Comments section](#-comments)

`jobs` - asynchronous processes<br>

> each process will starts a virtual machine and has:<br>

Reviewer: ```suggestion
each process will start a virtual machine with:<br>
```

`node` - node.js version<br>
`installCommand` - command to install dependencies<br>
`buildCommand` - the command to build the project

> `steps` - array of checks in within a separate process<br>

Reviewer: ```suggestion
`steps` - array of checks within a separate process<br>
```
